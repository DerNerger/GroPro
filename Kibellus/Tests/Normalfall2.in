Schwanz
Katze
Haustuer
Keller
Teller
Mutter