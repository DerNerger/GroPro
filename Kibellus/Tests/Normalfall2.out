Eingelesene Woerter:
SCHWANZ, KATZE, HAUSTUER, KELLER, TELLER, MUTTER

** Raetsel nicht versteckt

    K  K 
SCHWANZE 
 MUTTERL 
    Z  L 
   TELLER
HAUSTUER 

** Raetsel versteckt

HNALKEQKX
SCHWANZEI
SMUTTERLT
KHGTZIZLQ
QADTELLER
HAUSTUERP

Kompaktheitsmass: 54